### ISSUE TITLE
Docker “--all-in-one” fails with ModuleNotFoundError: No module named ‘tqdm’

### ISSUE THREAD

**Issue by cmdicely** [ggerganov/llama.cpp#289]
On Win 10
```
>  docker run -v /llama/models:/models ghcr.io/ggerganov/llama.cpp:full –all-in-one “/models/” 7B
Downloading model…
Traceback (most recent call last):
  File “/app/./download-pth.py”, line 3, in <module>
    from tqdm import tqdm
ModuleNotFoundError: No module named ‘tqdm’
```

**Comment by bvajk** [comment]
My guess is putting tqdm at the end of .devops/full.Dockerfile might solve the issue but I don't how to rebuild the image ;/

**Comment by gjmulder** [comment]
Please confirm that the problem is fixed after the pull request is merged

**Comment by bvajk** [comment]
Why would you merge before knowing it fixes the problem in the first place?

**Comment by domifi** [comment]
I have the same problem and am happy to test. I'm just not sure how to build the docker image locally

**Comment by bvajk** [comment]
> I have the same problem and am happy to test. I'm just not sure how to build the docker image locally

Me too, the actions on [the PR](https://github.com/ggerganov/llama.cpp/pull/293) suggest that the change has been pushed to docker, but I can't figure out how to pull it

**Comment by domifi** [comment]
Ok, I think I got it and it seems to be working.
I believe the [action](https://github.com/ggerganov/llama.cpp/actions/runs/4460613144/jobs/7833885640?pr=293)  doesn't actually push the image anywhere and is only for testing whether the build even succeeds. Just after the build command the log says `WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load`

I was able to build the image locally with: `docker build -f .devops/full.Dockerfile .`
